### DISCUSSION TITLE
Adonis multi tenancy

### DISCUSSION THREAD

**Discussion by Jon710** [adonisjs/core#2741] [Help]
Has anyone implemented multi tenancy with Adonis?

**Comment by guntribam** [comment]
Please look #1587

**Reply by Jon710** [reply]
Thanks, I'll take a look in it.

**Comment by Salman-Kashfy** [comment]
> Has anyone implemented multi tenancy with Adonis?

I have created a complete solution for this in Adonis 5. Here is the github repo and a step by step guide in the git description

https://github.com/Salman-Kashfy/Multitenancy-AdonisJS-5/

**Reply by britzdylan** [reply]
Good Work

**Reply by Jon710** [reply]
Nice! Maybe I could use this solution with Express, since I'm using this framework right now
